openc3/bin/openc3cli

```diff
@@ -569,14 +579,32 @@ if not ARGV[0].nil? # argument(s) given
     ENV['OPENC3_NO_STORE'] = '1'
     filename = ARGV[1]
     filename = 'bridge.txt' unless filename
-    bridge = OpenC3::Bridge.new(filename)
-    begin
-      while true
-        sleep(1)
+    params = ARGV[2..-1]
+    params = [] unless params
+    run_bridge(filename, params)
+
+  when 'bridgegem'
+    ENV['OPENC3_NO_STORE'] = '1'
+    filename = nil
+    gem_name = ARGV[1]
+    Gem::Specification.each do |s|
+      if s.name == gem_name
+        if Array === Gem.path
+          Gem.path.each do |gem_path|
+            filename = File.join(gem_path, 'gems', "#{s.name}-#{s.version}", 'bridge.txt')
+            puts "Trying #{filename}"
+            break if File.exist?(filename)
+          end
+        else
+          filename = File.join(Gem.path, 'gems', "#{s.name}-#{s.version}", 'bridge.txt')
+        end
```

Comment:
Is this effectively looking for the last gem spec? Is that the latest?

Reply:
Good catch.  I didn't realize it returns all versions.

Reply:
It appears to always return the newest version first, but I'll add a comment to that effect.